### PR TITLE
docs: adding cbroti requirement for custom caddy build.

### PR DESCRIPTION
**Pull request by StephenMiracle**
It was failing to build and run correctly without this module. The caddyfile shows the br encoding.

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -154,8 +155,8 @@ USER ${USER}
 
 The Docker images are built:
 
-* when a new release is tagged
-* daily at 4am UTC, if new versions of the official PHP images are available
+- when a new release is tagged
+- daily at 4am UTC, if new versions of the official PHP images are available
```
Comment (dunglas) [inline docs/docker.md:159]: Can you revert these changes please?

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by dunglas** [comment]
Good catch. We need to use Go build directives to not have a hard dependency on cbrotli. I'll do that but let's update the docs in the meantime.

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks!

**Comment by StephenMiracle** [comment]
Ah. Thanks for updating back the asterisks. I just went to change. VS Code autoformatter. Thanks.

**Pull request merged**